src/settings.ts

```diff
@@ -34,6 +41,13 @@ export function resolveSettings(raw: Partial<Settings>): Settings {
         ['codecov.decorations.lineCoverage']: !!raw['codecov.decorations.lineCoverage'],
         ['codecov.decorations.lineHitCounts']: !!raw['codecov.decorations.lineHitCounts'],
         ['codecov.endpoints']: [resolveEndpoint(raw['codecov.endpoints'])],
+        ['codecov.fileDecorations.low']:
+            typeof raw['codecov.fileDecorations.low'] === 'number' ? raw['codecov.fileDecorations.low'] : 70,
+        ['codecov.fileDecorations.high']:
+            typeof raw['codecov.fileDecorations.high'] === 'number' ? raw['codecov.fileDecorations.high'] : 85,
+        ['codecov.fileDecorations.optimum']:
+            typeof raw['codecov.fileDecorations.optimum'] === 'number' ? raw['codecov.fileDecorations.optimum'] : 100,
+        ['codecov.fileDecorations.show']: raw['codecov.fileDecorations.show'] || false,
```

Comment:
I saw that you were using the `??` operator here previously. I'm curious, why did you switch to this `typeof` expression instead?